src/Lykke.Sdk/LykkeApplicationBuilderExtensions.cs

```diff
@@ -29,8 +29,8 @@ public static void UseLykkeConfiguration(this IApplicationBuilder app)
         /// Configure Lykke service.
         /// </summary>
         /// <param name="app">IApplicationBuilder implementation.</param>
-        /// <param name="defaultErrorHandler">Default error handler.</param>
-        public static void UseLykkeConfiguration(this IApplicationBuilder app, CreateErrorResponse defaultErrorHandler)
+        /// <param name="configurationOptions">Configuration options - <see cref="LykkeConfigurationOptions"/>.</param>
+        public static void UseLykkeConfiguration(this IApplicationBuilder app, LykkeConfigurationOptions configurationOptions)
```

Comment:
Better to use Action<LykkeConfigurationOptions> this will allow us to always have options object, make some default configuration before options will be passed to the application and make ctor of LykkeConfigurationOptions internal.